src/app/static/src/app/components/plots/utils.ts

```diff
@@ -44,73 +44,35 @@ export const colorFunctionFromName = function (name: string) {
     return result;
 };
 
-export const getIndicatorRanges = function (data: any,
-                                            indicatorsMeta: ChoroplethIndicatorMetadata[],
-                                            filters: Filter[] | null = null,
-                                            selectedFilterValues: Dict<FilterOption[]> | null = null,
-                                            selectedAreaIds: string[] | null = null): Dict<NumericRange> {
-    const result = {} as Dict<NumericRange>;
-    iterateDataValues(data, indicatorsMeta, selectedAreaIds, filters, selectedFilterValues,
+export const getIndicatorRange = function (data: any,
+                                           indicatorMeta: ChoroplethIndicatorMetadata,
+                                           filters: Filter[] | null = null,
+                                           selectedFilterValues: Dict<FilterOption[]> | null = null,
+                                           selectedAreaIds: string[] | null = null): NumericRange {
+    let result = {} as NumericRange;
```

Comment:
I think we want to roundRange the full dynamic as well as filtered dynamic ranges. It's getting applied in getDyanamicFilteredColourRange, which calls this method, but maybe it should be applied to the result here before returning, so it applies to full ranges too.

Reply:
Ah cool, I've just moved the rounding logic into `getIndicatorRange` and removed the duplicate `getDyanamicFilteredColourRange` method - does that seem right to you? There's no reason not to always round the former right?